src/main/java/net/openhft/chronicle/core/Maths.java

```diff
@@ -164,7 +163,7 @@ public static double round1(double d) {
         if (d < 0)
             return -round1(-d);
         final double factor = 1e1;
-        if (!(d <= WHOLE_NUMBER / factor)) // to handle NaN
```

Comment:
this changes behaviour - the comment explains why

Reply:
Oooh good pickup

Reply:
We should add a test for NaN for one of these functions to capture this

Reply:
Wow. This was clumsy of me to miss. Thanks, @JerryShea for saving my bacon...